app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```diff
@@ -143,7 +145,17 @@ public void possibleResultPoints(List<ResultPoint> resultPoints) {
     };
 
     private void findProduct(String lastText) {
-        client.getFullProductByBarcodeSingle(lastText)
+        String s = "off:off";
```

Comment:
@stephanegigandet What does it do on the live server if we try a basic auth ?
@huzaifaiftikhar perhaps bind the behaviour behind some key ?

Reply:
This will be removed when finalizing the PR for the production server. Since we don't need any authentication on thr production server.